twilio/twilio_data_mappings.bal

```diff
@@ -44,69 +44,69 @@ function mapJsonToVoiceCallResponse(json jsonPayload) returns VoiceCallResponse
 }
 
 function mapJsonToAuthyAppDetailsResponse(json jsonPayload) returns AuthyAppDetailsResponse {
-    AuthyAppDetailsResponse authyAppDetailsResponse;
+    AuthyAppDetailsResponse authyAppDetailsResponse = {};
     authyAppDetailsResponse.appId = jsonPayload.app.app_id.toString();
     authyAppDetailsResponse.name = jsonPayload.app.name.toString();
     authyAppDetailsResponse.plan = jsonPayload.app.plan.toString();
-    authyAppDetailsResponse.isSmsEnabled = <boolean>jsonPayload.app.sms_enabled.toString();
-    authyAppDetailsResponse.isPhoneCallsEnabled = <boolean>jsonPayload.app.phone_calls_enabled.toString();
-    authyAppDetailsResponse.isOnetouchEnabled = <boolean>jsonPayload.app.onetouch_enabled.toString();
+    authyAppDetailsResponse.isSmsEnabled = boolean.create(jsonPayload.app.sms_enabled.toString());
```

Comment:
`.create(...)` function is changed as `.convert(...)`
Please replace in all the places.